udata/core/organization/api_fields.py

```diff
@@ -71,7 +71,10 @@
     'page': fields.UrlFor(
         'organizations.show', lambda o: {'org': o},
         description='The organization page URL', readonly=True),
-    'logo': fields.ImageField(description='The organization logo URLs'),
+    'logo': fields.ImageField(original=True,
+        description='The organization logo URL'),
+    'logo_thumbnail': fields.ImageField(size=150,
```

Comment:
Why `150` given that we have `100` above for the logo of `org_ref_fields`? Magic number => descriptive var?